homeassistant/components/sunweg/sensor_types/sensor_entity_description.py

```diff
@@ -0,0 +1,23 @@
+"""Sensor Entity Description for the SunWEG integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import SensorEntityDescription
+
+
+@dataclass
+class SunWEGRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    api_key: str
```

Comment:
Maybe rename this to api_variable or something similar so that it is a bit clearer. At first glance I thought it was a key to connect to the api rather than a variable to access certain data in the api.